browser/src/index.ts

```diff
@@ -0,0 +1,103 @@
+
+function unpackU64toU8(n: BigInt): Uint8Array {
+    //@ts-ignore
+    return new Uint8Array(new BigInt64Array([n]).buffer)
+}
+
+function packU8toU64(bytes: Uint8Array): BigInt {
+    return (new BigInt64Array(bytes.buffer))[0]
```

Comment:
maybe a better way to pack and unpack bytes. But my bit shifting and mask code was buggy and I think in danger of overflows. I assume this is safe but could use some scrutiny